build_library/build_image_util.sh

```diff
@@ -154,6 +154,15 @@ write_packages() {
     fi
 }
 
+extract_docs() {
+    local root_fs_dir="$1"
+
+    info "Extracting docs"
+    tar --create --bzip --file="${BUILD_DIR}/docs.tar" \
```

Comment:
Missing a .bz2 extension. Also I think tar is the one case where I find long args more confusing than the short. This should do fine: `tar -xf "${BUILD_DIR}/doc.tar.bz2" -C "${root_fs_dir}/usr/share/coreos" doc`

That has the added bonus of including a top level directory in the tarball.